Sming/building.md

```diff
@@ -0,0 +1,138 @@
+# Sming makefiles

```

Comment:
There are newly added files that have Windows line endings. Please, use `dos2unix` or some other tool to convert them. I saw that issue in this file - building.md, other md files and some of the new mk files.
![image](https://user-images.githubusercontent.com/247452/56966091-c48a3300-6b5e-11e9-8a3e-1928ce2ea25a.png)